tasks/managed/collect-data/collect-data.yaml

```
@@ -125,6 +125,38 @@ spec:
         echo -e "\nFetching fbcFragment"
         jq -cr '.components[0].containerImage' < "$(workspaces.data.path)/$SNAPSHOTSPEC_PATH" | tr -d "\n" \
           | tee "$(results.fbcFragment.path)"
+  
+        echo -e "\nGenerating collectors data"
+        collectors_status=$(get-resource "release" "${RELEASE}" "{.status.collectors}")
+        echo "***collectors status"
+        echo "${collectors_status}"
+        echo "***"
+
+        collectors_result=$(jq -c '
+          def deepmerge(a; b):
+            reduce b[] as $item (a;
+            reduce ($item | keys_unsorted[]) as $key (.;
+            $item[$key] as $val | ($val | type) as $type | .[$key] = if ($type == "object") then
+            deepmerge({}; [if .[$key] == null then {} else .[$key] end, $val])
+            elif ($type == "array") then
+            (.[$key] + $val | unique)
+            else
+            $val
+            end)
+            );
+        
+          # Ensure we safely handle missing collectors
+          (.? // {}) as $collectors |
+        
+          # Flatten and combine the managed and tenant sections
+          [($collectors.managed? // {} | to_entries | map(.value)) +
+           ($collectors.tenant? // {} | to_entries | map(.value))] |
+           flatten |
+           deepmerge({}; .)
+        ' <<< "${collectors_status}")
+        echo "***collectors"
+        echo "${collectors_result}"
```

You generate this with `jq -c` meaning it will be condensed to one line - not really readable. So perhaps do this?
```suggestion
        echo "$(jq <<< "$collectors_result")"
```

---

fixed. thanks